### ISSUE TITLE
Expecta uses NS_DESIGNATED_INITIALIZER which is undefined for SDKs before OSX10.10

### ISSUE THREAD

**Issue by jimix** [cloudant/CDTDatastore#160]
Using [xctool](https://github.com/facebook/xctool), which just wraps for `xcodebuild`
When building for OSX10.9:
```shell
$ (cd Tests; pod install)
$ xctool -workspace CDTDatastore.xcworkspace -scheme "Tests OSX" -sdk macosx10.9 test
...
In file included from /Users/jimix/work/iOS/CoreData/CDTDatastore/Tests/Pods/Expecta/Expecta/Matchers/EXPMatchers+beCloseTo.m:1:
In file included from /Users/jimix/work/iOS/CoreData/CDTDatastore/Tests/Pods/Expecta/Expecta/Matchers/EXPMatchers+beCloseTo.h:1:
In file included from /Users/jimix/work/iOS/CoreData/CDTDatastore/Tests/Pods/Expecta/Expecta/Expecta.h:10:
In file included from /Users/jimix/work/iOS/CoreData/CDTDatastore/Tests/Pods/Headers/Private/Expecta/ExpectaSupport.h:1:
/Users/jimix/work/iOS/CoreData/CDTDatastore/Tests/Pods/Headers/Private/Expecta/EXPExpect.h:31:159: error: expected ':'
- (instancetype)initWithActualBlock:(id)actualBlock testCase:(id)testCase lineNumber:(int)lineNumber fileName:(const char *)fileName NS_DESIGNATED_INITIALIZER;
                                                                                                                                                              ^
                                                                                                                                                              :
/Users/jimix/work/iOS/CoreData/CDTDatastore/Tests/Pods/Headers/Private/Expecta/EXPExpect.h:43:110: error: expected ':'
- (instancetype)initWithExpectation:(EXPExpect *)expectation selector:(SEL)selector NS_DESIGNATED_INITIALIZER;
```

I'll follow up on any issues with pre iOS8.

**Comment by indisoluble** [comment]
Fixed in PR #144 (commit 0bbbdb7)

**Comment by rhyshort** [comment]
We now only support building on Xcode 6.4, which only runs on Yoesmite.
